src/core/Table/Table.tsx

```diff
@@ -69,6 +71,21 @@ export type TableProps<
    * Must be memoized.
    */
   onSort?: (state: TableState<T>) => void;
+  /**
+   * Callback function when scroll reaches bottom. Can be used for lazy-loading the data.
+   * If you want to use it in older browsers e.g. IE, then you need to have IntersectionObserver polyfill.
+   */
+  onBottomReached?: () => void;
+  /**
+   * Callback function when row is viewport.
+   * If you want to use it in older browsers e.g. IE, then you need to have IntersectionObserver polyfill.
+   */
+  onRowIntersection?: (rowData: T) => void;
+  /**
+   * Margin when row is considered to be already in view. Used for `onBottomReached` and `onRowIntersection`.
+   * @default 300
+   */
+  intersectionMarginPx?: number;
```

Comment:
No need for px in the name. Add it to the comment instead.

Reply:
Changed.